_analyzers/tokenizers/index.md

```diff
@@ -0,0 +1,61 @@
+`classic` | - Parses strings into tokens on: <br> &emsp; - punctuation characters that are followed by a white space character <br> &emsp; - hyphens if the term does not contain numbers <br> - Removes punctuation <br>  - Leaves URLs and email addresses as single terms | `Part number PA-35234, single-use product (128.32)` <br>becomes<br> [`Part`, `number`, `PA-35234`, `single`, `use`, `product`, `128.32`]
+`thai` | - Parses Thai text into terms | `สวัสดีและยินดีต` <br>becomes<br> [`สวัสด`, `และ`, `ยินดี`, `ต`] 
+
+### Partial word tokenizers
+
+Partial word tokenizers parse text into words and generate fragments of those words for partial word matching.
+
+Tokenizer | Description | Example
+:--- | :--- | :---
+`ngram`| - Parses strings into words on specified characters (for example, punctuation or white space characters) and generates n-grams of each word | `My repo` <br>becomes<br> [`M`, `My`, `y`, `y `, <code>&nbsp;</code>, <code>&nbsp;r</code>, `r`, `re`, `e`, `ep`, `p`, `po`, `o`] because the default n-gram length is 1--2 characters. 
+`edge_ngram` | - Parses strings into words on specified characters (for example, punctuation or white space characters) and generates edge n-grams of each word (n-grams that start at the beginning of the word) | `My repo` <br>becomes<br> [`M`, `My`] because the default n-gram length is 1--2 characters. 
+
+### Structured text tokenizers
+
+Structured text tokenizers parse structured text, such as identifiers, email addresses, paths, or zip codes.
+
+Tokenizer | Description | Example
+:--- | :--- | :---
+`keyword` | - No-op tokenizer <br> - Outputs the entire string unchanged <br> - Can be combined with token filters like lowercase to normalize terms | `My repo` <br>becomes<br> `My repo`
+`pattern` | - Uses a regular expression pattern to parse text into terms on a word separator or to capture matching text as terms <br> - Uses [Java regular expressions](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) | `https://opensearch.org/forum` <br>becomes<br> [`https`, `opensearch`, `org`, `forum`] because by default the tokenizer splits terms at word boundaries (`\W+`). Can be configured with a regex pattern.
+`simple_pattern` | - Uses a regular expression pattern to return matching text as terms <br>  - Uses [Lucene regular expressions](https://lucene.apache.org/core/8_7_0/core/org/apache/lucene/util/automaton/RegExp.html)  <br> - Faster than the `pattern` tokenizer because it uses a subset of the `pattern` tokenizer regular expressions |  Returns an empty array by default. Must be configured with a pattern because the pattern defaults to an empty string.
```

Comment:
Ideally, the phrases in the table would either all use punctuation or not use punctuation. You have a few entries in the right-hand column that use periods, whereas the rest of the table(s) do not use punctuation. Your choice as to how to make consistent.